docs/terraform-provider-documentation

```diff
@@ -0,0 +1,3 @@
+# Terraform provider documentation
+
+Save time on managing Airbyte Cloud and collaborate on Airbyte configuration changes with your teammates. Airbyte's Terraform provider is based on our new [Airbyte API}(! The Terraform provider is available for Airbyte Cloud customers today (& OSS in Q3). Check out our guide for getting started with our Terraform provider(Save time on managing Airbyte Cloud and collaborate on Airbyte configuration changes with your teammates. Airbyte's Terraform provider is based on our new [Airbyte API](https://reference.airbyte.com/reference/start)! The Terraform provider is available for Airbyte Cloud customers today (& OSS in Q3). Check out our guide for [getting started with Airbyte's Terraform provider](https://reference.airbyte.com/reference/using-the-terraform-provider).
```

Comment:
looks like this part is unintentional `[Airbyte API}(!` and there same doc repeats itself

Reply:
Whoops! Thanks